dev/slack-notifications-for-github-actions.md

```diff
@@ -0,0 +1,21 @@
+# Slack notifications for GitHub Actions
+
+Max Horn, Chris Jefferson, Alexander Konovalov, Sergio Siccha, and Wilf Wilson
+have access to the app `GAP GH Actions Notifications`. If you have access, then
```

Comment:
Good idea to document this somewhere; should also (instead?) be on https://github.com/gap-system/gap/wiki/Developer-resources (which in turn also needs to be updated). Or perhaps that file in turn should be part of the GAP repo.

@alex-konovalov @ChrisJefferson thoughts?

Reply:
I just updated the Developer-resources Wiki page a bit, but I am sure I missed things

Reply:
I've removed this file and added an according wiki page.

Reply:
And just added links to that wiki page.